### PR TITLE
Fix installation

### PR DESCRIPTION
**Pull request by AleksanderPasiut**

### CODE REVIEW

File: capdExt/CMakeLists.txt [hunk 1]
```diff
@@ -12,7 +12,7 @@ ExternalProject_Add(
     SOURCE_DIR ${CMAKE_CURRENT_BINARY_DIR}/filibsrc
     BUILD_IN_SOURCE TRUE
     PATCH_COMMAND cp -r ${CMAKE_CURRENT_SOURCE_DIR}/filibsrc ${CMAKE_CURRENT_BINARY_DIR}
-    CONFIGURE_COMMAND ${CMAKE_CURRENT_BINARY_DIR}/filibsrc/configure --prefix=${CMAKE_CURRENT_BINARY_DIR}/filibsrc-install
+    CONFIGURE_COMMAND autoreconf -i && ${CMAKE_CURRENT_BINARY_DIR}/filibsrc/configure --prefix=${CMAKE_CURRENT_BINARY_DIR}/filibsrc-install
```
Comment (AleksanderPasiut) [inline capdExt/CMakeLists.txt:15]: Without autoreconf filib build has various issues.

File: capdExt/filibsrc/libprim/Makefile.am [hunk 2]
```diff
@@ -3,7 +3,7 @@ AUTOMAKE_OPTIONS = subdir-objects
 EXTRA_DIST = 
 SUBDIRS =
 
-noinst_LTLIBRARIES = libprim.la
+lib_LTLIBRARIES = libprim.la
```
Comment (AleksanderPasiut) [inline capdExt/filibsrc/libprim/Makefile.am:6]: `noinst_` prevents filib library from being installed => fixed

File: capdMake/libcapd/CMakeLists.txt [hunk 3]
```diff
@@ -7,11 +7,11 @@ string(REPLACE ";" " " CAPD_LIBS "${CAPD_LIBS}")
 set(PKG_CONFIG_PATH ${CMAKE_INSTALL_PREFIX}/lib/pkgconfig)
 configure_file(
     ${CMAKE_CURRENT_SOURCE_DIR}/capd-config.in
-    ${CMAKE_CURRENT_BINARY_DIR}/bin/capd-config
+    ${CMAKE_CURRENT_BINARY_DIR}/capd-config
```
Comment (AleksanderPasiut) [inline capdMake/libcapd/CMakeLists.txt:10]: removed intermediate `bin` directory here to make it simpler

File: capdMake/libcapd/CMakeLists.txt [hunk 4]
```diff
@@ -21,6 +21,6 @@ configure_file(
     @ONLY)
 
 install(
-  FILES ${CMAKE_CURRENT_BINARY_DIR}/bin/capd.pc
-  PERMISSIONS OWNER_EXECUTE OWNER_READ
-  DESTINATION ${PKG_CONFIG_PATH}/capd.pc)
\ No newline at end of file
+  FILES ${CMAKE_CURRENT_BINARY_DIR}/capd.pc
+  PERMISSIONS OWNER_READ
```
Comment (AleksanderPasiut) [inline capdMake/libcapd/CMakeLists.txt:25]: `OWNER_EXECUTE` permission is not necessary

File: capdMake/libcapd/CMakeLists.txt [hunk 5]
```diff
@@ -21,6 +21,6 @@ configure_file(
...
+  DESTINATION ${PKG_CONFIG_PATH})
```
Comment (AleksanderPasiut) [inline capdMake/libcapd/CMakeLists.txt:26]: install takes only destination directory, not the target name => fixed

File: capdMake/libcapd/capd-gui-config-dev.in
Comment (AleksanderPasiut) [inline capdMake/libcapd/capd-gui-config-dev.in:1]: krak component removed, so `capd-gui` scripts are no longer necessary

**Comment by dbwilczak** [comment]
Examples and tests still do not compile if filib is not installed in the system. 
[100%] Linking CXX executable DPoincareMapMaskTest
/usr/bin/ld: cannot find -lprim: No such file or directory
collect2: error: ld returned 1 exit status

I would suggest to:
- add just few files from filib to compilation system of CAPD instead of ./configure of filib
- create two versions of capd-config scripts: first for installation in the system and second for using the CAPD library without installation (as it was in the precious version).

**Pull request merged**